content/en/data_jobs/databricks.md

```diff
@@ -71,6 +71,16 @@ This setup does not work on Databricks clusters in <strong>Standard</strong> (fo
 1. Click **Save Databricks Workspace** at the bottom of the browser window.
    {{< img src="data_jobs/databricks/configure-data-jobs-monitoring-existing.png" alt="In the Datadog-Databricks integration tile, Datadog Agent Setup for a Databricks workspace already added to the integration. Datadog can install and manage a global init script." style="width:100%;" >}}
 
+Optionally, you can add custom host tags to your Databricks clusters by configuring the following environment variable in the Advanced Configuration section of your cluster in the Databricks UI or as [Spark env vars][2] with the Databricks API:
+
+| Variable                 | Description                                                                                                                                                      | Default |
+|--------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
+| DD_TAGS                  | Add host tags to Databricks clusters. Comma or space separated key:value pairs. Follow [Datadog tag conventions][1]. Example: `env:staging,team:data_engineering` |         |
```

Comment:
@aboitreaud do we actually want this as the method for showing how to set `env`? Isn't that `DD_ENV` ?